atom/browser/native_window_mac.mm

```diff
@@ -444,6 +468,36 @@ - (NSView*)frameView {
   return [[self contentView] superview];
 }
 
+// Quicklook methods
+
+- (BOOL)acceptsPreviewPanelControl:(QLPreviewPanel *)panel {
+  return YES;
+}
+
+- (void)beginPreviewPanelControl:(QLPreviewPanel *)panel {
+  panel.delegate = self;
+  panel.dataSource = self;
+}
+
+- (void)endPreviewPanelControl:(QLPreviewPanel *)panel {
+  panel.delegate = nil;
+  panel.dataSource = nil;
+}
+
+- (NSInteger)numberOfPreviewItemsInPreviewPanel:(QLPreviewPanel *)panel {
+  return 1;
+}
+
+- (id <QLPreviewItem>)previewPanel:(QLPreviewPanel *)panel previewItemAtIndex:(NSInteger)index {
+  return [self quickLookItem];
+}
+
+- (void)previewFileAtPath:(NSString *)filepath  withName:(NSString *) name {
+  NSURL * url = [[NSURL alloc] initFileURLWithPath:filepath];
```

Comment:
It looks like this file isn't using ARC, so there should probably be an `autorelease` here, and on the next line where the preview item is built (because `self.quickLookItem` specifies it's behavior as `retain`, so it'll hold on to the object.)

I might be wrong—if you add `autorelease` and it throws a compiler error, ARC is on and you don't need it :-)